.circleci/config.yml

```diff
@@ -11,23 +11,23 @@ install_and_test: &install_and_test
         command: npm test
 
 jobs:
-  test_node4:
+  test_node8:
     docker:
-      - image: node:4
+      - image: node:8
     <<: *install_and_test
-  test_node6:
+  test_node10:
     docker:
-      - image: node:6
+      - image: node:10
     <<: *install_and_test
-  test_node8:
+  test_node11:
     docker:
-      - image: node:8
+      - image: node:11
     <<: *install_and_test
 
 workflows:
   version: 2
   test:
     jobs:
-      - test_node4
-      - test_node6
-      - test_node8
\ No newline at end of file
+      - test_node8
+      - test_node10
+      - test_node11
```

Comment:
probably want to include v12 as well

Reply:
Thanks for the review. For now, I'm just matching what the client libraries do, so that the compatibility boundaries are equal.